src/config.js

```diff
@@ -62,7 +62,8 @@ class Dirs {
     d = d || path.join(this.home, category === 'data' ? '.local/share' : '.' + category)
     if (this.windows) d = process.env.LOCALAPPDATA || d
     d = process.env.XDG_DATA_HOME || d
-    d = path.join(d, this._config.name)
+    let n = this._config.name === 'heroku-cli' ? 'heroku' : this._config.name
```

Comment:
@dickeyxxx that is what I was thinking as well and `dirname` sounds like a good name for it and I was thinking it would be in the [`cli-engine`](https://github.com/heroku/cli/blob/a2194a07e030a3fb2206cee61413e4316b730cec/package.json#L10) definition